CONTRIBUTING.md

```diff
@@ -136,7 +134,7 @@ This project written in ES2016+ syntax so, we need to transpile it before use.
 So run the following command:
 
 ```sh
-npm run dev
+yarn run dev
```

Comment:
`yarn dev`

Reply:
Ok, I just wrote verbose versions of commands so that it's easilly mapped to npm. If we go all the way yarn, I'll use shorthands